app/views/schools/meters/_form.html.erb

```diff
@@ -23,24 +23,38 @@
       </div>
     <% end %>
   </div>
+
   <div class="form-group">
+
     <p class="alert alert-warning">Experimental features for N3RGY integration</p>
-    <div class="custom-control custom-checkbox">
-      <%= form.check_box :dcc_meter, class: 'custom-control-input'  %>
-      <%= form.label :dcc_meter, 'DCC Smart Meter', class: "custom-control-label"  %>
-    </div>
-    <div class="custom-control custom-checkbox">
-      <%= form.check_box :consent_granted, class: 'custom-control-input'  %>
-      <%= form.label :consent_granted, 'Consent granted', class: "custom-control-label"  %>
-    </div>
+
+    <% if show_dcc_fields %>
+
+      <% if meter_is_dcc %>
+        <p class="alert alert-success">This meter is available in the DCC</p>
```

Comment:
Suggest a wording tweak here. Based on what we know now, I think "available in DCC" is basically anything where we can do a Read Inventory. This status call is now showing "available via N3RGY". Might make it clearer about about we are showing?